core/orchestrator.go

```diff
@@ -90,8 +91,8 @@ func (orch *orchestrator) ServeTranscoder(stream net.Transcoder_RegisterTranscod
 	orch.node.serveTranscoder(stream, capacity)
 }
 
-func (orch *orchestrator) TranscoderResults(tcId int64, res *RemoteTranscoderResult) {
-	orch.node.TranscoderManager.transcoderResults(tcId, res)
+func (orch *orchestrator) TranscoderResults(tcID int64, res *RemoteTranscoderResult) {
```

Comment:
With these changes, does the `core` package pass the CircleCI linter? Is there a way to lint specific files so we don't regress on this?

Reply:
We could lint whole package or separate file. `orchestrator.go` still has many linter errors. I've tried to fix them, but stopped at one that will lead to big changes:
`func NewOrchestrator(n *LivepeerNode) *orchestrator` returns private struct `orchestrator`.
It should be either changed to be public (not good), or we should return interface here - but this interface (`type Orchestrator interface`) is defined in `server` package, and to use it in `core` we should move it to some other package, probably to introduce `model` package.